ci/test/04_install.sh

```diff
@@ -12,6 +12,7 @@ fi
 
 if [ "$CI_OS_NAME" == "macos" ]; then
   sudo -H pip3 install --upgrade pip
+  # shellcheck disable=SC2086
```

Comment:
What is the idea of disabling these specific checks in some places? Is the idea that these are temporary and need to be fixed later? (but too much hassle to fix in thir PS?)

Reply:
> What is the idea of disabling these specific checks in some places?

```bash
$ PIP_PACKAGES="zmq lief"
$ pip3 install --user "$PIP_PACKAGES"
ERROR: Invalid requirement: 'zmq lief'
```

Reply:
Looks like that. And that code requires `# shellcheck disable=SC2086`.

Reply:
Also see https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions

Reply:
This works:
```bash
$ PIP_PACKAGES=(zmq lief)
$ pip3 install --user "${PIP_PACKAGES[@]}"
```

Reply:
Using arrays sounds like a good idea to me. At least for scripts like this that explicitly need `bash`.